README.md

```diff
@@ -1,113 +1,52 @@
-# IntegriTEE Parachain:
+# Integritee Parachain
 
-This is the repository to run integritee as a parachain in the rococo-v1 testnet. It is forked from the [Cumulus](https://github.com/paritytech/cumulus) repository and only adds the SubstraTEE pallet and configuration.
+This is the repository to run Integritee as a parachain on Kusama and Rococo. It is forked from the [Cumulus](https://github.com/paritytech/cumulus) repository.
 
 ## Launch a local setup including a Relay Chain and a Parachain
 
+### Polkadot-launch
+
 Quick: see [polkadot-launch](https://github.com/paritytech/polkadot-launch.git)
 ```
-node ../polkadot-launch/dist/cli.js launch-kusama-local-with-shell.json
+node ../polkadot-launch/dist/cli.js launch-rococo-local-with-integritee.json
 ```
-
-
-### Manually Launch Local Rococo Testnet
-
-```bash
-# Compile Polkadot with the real overseer feature
-git clone https://github.com/paritytech/polkadot
-git checkout <release tag>  // release tag that matches the branch id in the polkadot-deps
-cargo build --release
-
-# Generate a raw chain spec
-./target/release/polkadot build-spec --chain rococo-local --disable-default-bootnode --raw > rococo-local-cfde.json
-
-# Alice
-./target/release/polkadot --chain rococo-local-cfde.json --alice --tmp
-
-# Bob (In a separate terminal)
-./target/release/polkadot --chain rococo-local-cfde.json --bob --tmp --port 30334
+or
 ```
-
-#### Launch the Parachain
-
-```bash
-# Compile
-git clone https://github.com/scs/integritee-parachain.git
-git checkout master
-cargo build --release
-
-# Export genesis state
-./target/release/integritee-collator export-genesis-state --chain integritee-local-dev > integritee-local-dev.state
-
-# Export genesis wasm
-./target/release/integritee-collator export-genesis-wasm --chain integritee-local-dev > integritee-local.wasm
-
-# Collator
-./target/release/integritee-collator --collator --tmp --chain integritee-rococo-local-dev --port 40335 --ws-port 9946 -- --execution wasm --chain ../polkadot/rococo-local-cfde.json --port 30337 --ws-port 9981
+polkadot-launch launch-rococo-local-with-integritee.json
 ```
+depending on how you installed `polkadot-launch`.
```

Comment:
I would prefer if we set a team standard here. Go with the second option but explain how to install it that way

Reply:
whereas "global" means "across all projects of the current user", not system-wide for all users. then it's ok as it does not demand special privileges
`
.yarn/bin/polkadot-launch`

Reply:
Yes, it's per user. At least if yarn has been installed for per-user. It didn't require any sudo command, in my case, neither for yarn nor for polkadot-launch.